packages/quick-start-browser/src/BrowseFiles.tsx

```diff
@@ -0,0 +1,51 @@
+import { Icon, Link, Text } from "@fluentui/react";
+import { DocumentIcon, UploadIcon } from "@fluentui/react-icons-mdl2";
+import { registerIcons } from "@fluentui/react/lib/Styling";
+import React from "react";
+import { useDropzone } from "react-dropzone";
+
+registerIcons({
```

Comment:
RegisterIcons is Fluent8. 
 "@fluentui/react"; is fluent 8

since this is new proj I would advise to use only fluent 9
you can get icons from react icons in the following way: <name><size><regular|filled> 
import { CheckmarkCircle20Regular, Delete20Regular, DismissCircle20Filled } from '@fluentui/react-icons';

you can see icons available here: 
https://www.figma.com/file/43oQOCD2164ExeSf5ajmou/Fluent-iconography?mode=dev